src/api/networkId.ts

```diff
@@ -0,0 +1,20 @@
+export enum LegacyNetwork {
+  celo = 'celo',
+  celoAlfajores = 'celoAlfajores',
+}
+
+export enum NetworkId {
+  'celo-mainnet' = 'celo-mainnet',
+  'celo-alfajores' = 'celo-alfajores',
+  'ethereum-mainnet' = 'ethereum-mainnet',
+  'ethereum-sepolia' = 'ethereum-sepolia',
+  'arbitrum-one' = 'arbitrum-one',
+  'arbitrum-sepolia' = 'arbitrum-sepolia',
+  'op-mainnet' = 'op-mainnet',
+  'op-sepolia' = 'op-sepolia',
```

Comment:
I was on the fence about it but went with op-mainnet/sepolia because it aligns better with how Optimism's docs talk about their chains https://docs.optimism.io/chain/getting-started

Reply:
I made this choice earlier when we first introduced optimism in other repos btw (not in this PR). Though it wouldn't be too late to change it unless you guys have released it since then